packages/cli/test/commands/migrate/config-ts.ember-addon.test.ts

```diff
@@ -0,0 +1,148 @@
+import { resolve } from 'node:path';
+import { readdirSync } from 'node:fs';
+import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest';
+import { readJSONSync, writeJSONSync } from 'fs-extra/esm';
+import { tsConfigTask } from '../../../src/commands/migrate/tasks/index.js';
+import { listrTaskRunner, createMigrateOptions } from '../../test-helpers/index.js';
+import { runTsConfig, createUserConfig } from '../../test-helpers/config-ts-test-utils.js';
+import { TSConfig } from '../../../src/types.js';
+import { UserConfig } from '../../../src/user-config.js';
+import { setGracefulCleanup } from 'tmp';
+import { addonScenarios, clean } from '@rehearsal/test-support';
+import { Scenario, PreparedApp } from 'scenario-tester';
+
+setGracefulCleanup();
+
+describe('Task: config-ts ember app', () => {
+  addonScenarios.forEachScenario((scenario: Scenario) => {
+    describe(scenario.name, () => {
+      let app: PreparedApp;
+      let output = '';
+
+      beforeEach(async () => {
+        app = await scenario.prepare();
+        clean(app.dir);
```

Comment:
this is cleaning beforeEach, however we need to clean afterEach instead (since the last test execution wont be cleaned with this implementation)

Reply:
switched to project